test/support.js

```diff
@@ -29,7 +29,7 @@ Sequelize.Promise.onPossiblyUnhandledRejection(function(e, promise) {
 Sequelize.Promise.longStackTraces();
 
 // shim all Sequelize methods for testing for correct `options.logging` passing
-if (!process.env.COVERAGE) supportShim(Sequelize);
+if (!process.env.COVERAGE && false) supportShim(Sequelize);
```

Comment:
@overlookmotel i forgot about this, i'd like to you take a look at some point, but the supportShim severely fucked up during a refactor of HasMany.